globals.go

```diff
@@ -1084,3 +1084,55 @@ const (
 	LogViewerHTML    LogViewer = "html"
 	LogViewerLobster LogViewer = "lobster"
 )
+
+// ContainerOS denotes the operating system of a running container. Only linux and windows are supported.
+type ContainerOS string
+
+const (
+	LinuxOS   = "linux"
```

Comment:
In order for these to be of type `ContainerOS`, you have to include the type declaration or else these will be counted as just plain strings:
```
const (
LinuxOS ContainerOS = "linux"
WindowsOS ContainerOS = "windows"
)
```